docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "System Table Pre-Requisites"
+date: 2024-01-23T16:27:22+05:30
+---
+
+Once all System Table Integration Pre-requisites are completed, please continue to [Deploy Overwatch]({{%relref "DeployOverwatch/"%}})
+section.
+
+This section will walk you through the steps necessary as a prerequisite for System Table Integration with Overwatch.
+
+* **Workspace** should be **UC enabled** and **System Table enabled**. Enabling system tables is pretty easy, 
```

Comment:
"Enabling system tables is pretty easy" -> "Enabling system tables is straightforward"

Reply:
done